### ISSUE TITLE
Training Compute-Optimal Large Language Models

### ISSUE THREAD

**Issue by jinmang2** [jiphyeonjeon/season3#24]
## 집현전 최신반 스터디
- 2022년 5월 15일 일요일 10시
- 진명훈님 전재영님 박동주님 발표
- 논문 링크: https://arxiv.org/abs/2203.15556

> ### Abstract
> We investigate the optimal model size and number of tokens for training a transformer language model under a given compute budget. We find that current large language models are significantly undertrained, a consequence of the recent focus on scaling language models whilst keeping the amount of training data constant. By training over \nummodels language models ranging from 70 million to over 16 billion parameters on 5 to 500 billion tokens, we find that for compute-optimal training, the model size and the number of training tokens should be scaled equally: for every doubling of model size the number of training tokens should also be doubled. We test this hypothesis by training a predicted compute-optimal model, \chinchilla, that uses the same compute budget as \gopher but with 70B parameters and 4× more more data. \chinchilla uniformly and significantly outperforms \Gopher (280B), GPT-3 (175B), Jurassic-1 (178B), and Megatron-Turing NLG (530B) on a large range of downstream evaluation tasks. This also means that \chinchilla uses substantially less compute for fine-tuning and inference, greatly facilitating downstream usage. As a highlight, \chinchilla reaches a state-of-the-art average accuracy of 67.5\% on the MMLU benchmark, greater than a 7\% improvement over \gopher.

**Comment by jinmang2** [comment]
- 발표 링크: https://www.youtube.com/watch?v=xHbOZC79Ag4&list=PL2tRglRS_GqivYoDuzLNo_HezOvh4Z1XN&index=7
